iris_grib/tests/unit/grib_phenom_translation/test_grib_phenom_translation.py

```diff
@@ -31,11 +31,12 @@
 import cf_units
 
 import iris_grib.grib_phenom_translation as gptx
+from iris_grib.grib_phenom_translation import _LookupTable
```

Comment:
Well, not so sure now...
I guess it makes it clear that we're diddling with private elements ?

I will remove if you prefer.